### PR TITLE
[deprecation] LPS-105417 portal-impl, part 3 | rebased

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

File: portal-impl/src/com/liferay/portlet/asset/service.xml [hunk 1]
```diff
@@ -101,7 +101,7 @@
 		<reference entity="AssetEntry" package-path="com.liferay.portlet.asset" />
 		<reference entity="AssetVocabulary" package-path="com.liferay.portlet.asset" />
 	</entity>
-	<entity deprecated="true" local-service="true" name="AssetCategoryProperty" remote-service="true">
+	<entity deprecated="true" local-service="true" name="AssetCategoryProperty" remote-service="false">
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portlet/asset/service.xml:104]: @dantewang @ChiZelin 
This entity has been deprecated, why not remove the entity? Are there any usages left?

Tina.

**Comment by tinatian** [comment]
@dantewang @ChiZelin 
There are so many changes in this commit https://github.com/tinatian/liferay-portal/pull/1658/commits/1a6821218bafcf22a4d2086b373ec90cebfcd6f4, can you separate it into different commit? at least two commit, one for removing the whole class, one for removing unused methods.

Please resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
@dantewang @ChiZelin I merged part of this pull, please see https://github.com/shuyangzhou/liferay-portal/pull/8674, and resend the left ones.

Thanks.
Tina.

File: portal-impl/src/com/liferay/portlet/asset/service.xml (hunk 1 above)
Comment (ChiZelin) [reply portal-impl/src/com/liferay/portlet/asset/service.xml:104]: @tinatian Yes, there are. AssetCategoryPropertyLocalServiceUtil is used by AssetUtil.

**Pull request closed without merging**